packages/trader/src/Modules/SmartChart/Components/Markers/__tests__/accumulators-profit-loss-tooltip.spec.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import AccumulatorsProfitLossTooltip from '../accumulators-profit-loss-tooltip.jsx';
+import AccumulatorsProfitLossTooltip from '../accumulators-profit-loss-tooltip';
+
+type TAccumulatorsProfitLossTooltip = React.ComponentProps<typeof AccumulatorsProfitLossTooltip>;
```

Comment:
@henry-deriv it's not necessary to declare a type for the component props here because the type AccumulatorsProfitLossTooltip is already known to TS when we import AccumulatorsProfitLossTooltip in this file. Or Do you get any TS error here? )

Reply:
I dont remember getting errors. I'll remove

Reply:
@henry-deriv it seems like we don't need to have a variable for this type and use it directly since we use it just once